gztarchiver/doc_scraper/v2/runner.py

```
@@ -0,0 +1,275 @@
+
+            while True:
+                payload = [{
+                    "apiEndpoint": api_endpoint,
+                    "page": current_page,
+                    "limit": page_size,
+                    "q": "",
+                    "search": "",
+                    "forwarded": {},
+                }]
+
+                response = requests.post(
```

Ruff (0.16.2)

[error] Probable use of `requests` call without timeout

(S113)